deploy/crds/enterprise.splunk.com_indexerclusters_crd.yaml

```diff
@@ -2233,6 +2233,9 @@ spec:
           description: IndexerClusterStatus defines the observed state of a Splunk
             Enterprise indexer cluster
           properties:
+            cluster_master_maintenance_mode:
+              description: Indicated whether cm is in maintenance mode
```

Comment:
A generic comment, even including smartstore, I think, being these are not really spec related, a better way maybe we all move it to a configMap (Or, if there is a better way to make it even opaque by read/writes to the DB, if there is any such option) so that we don't pollute the spec going forward.

Reply:
Agreed subba, let's discuss if there is a better way to maintain state with DB read/write.